backend/app/api/auth.py

```
@@ -1,48 +1,154 @@
-    # 사용자 찾기
-    user = users_collection.find_one({"email": email})
-    if not user or not verify_password(password, user["password"]):
-        raise HTTPException(status_code=401, detail="이메일 또는 비밀번호가 잘못되었습니다")
-    
+@router.post("/dev-login")
+async def dev_login():
+    """개발용 자동 로그인 API - 테스트 사용자 자동 생성 및 로그인"""
+    # 테스트 사용자 정보
+    test_username = "testuser"
+    test_email = "test@example.com"
+    test_password = "password123"
```

Ruff (0.14.5)

Possible hardcoded password assigned to: "test_password"

(S105)